awx/ui_next/README.md

```diff
@@ -27,3 +27,14 @@ To run a single test (in this case the login page test):
 * `npm test -- src/screens/Login/Login.test.jsx`
 
 **note:** Once the test watcher is up and running you can hit `a` to run all the tests
+
+
+## Container
+
+To run the dev server in a container:
```

Comment:
Could you provide some short context about why you'd want to do this (or not)?

Reply:
@jlmitch5 The benefit is for environments where it's important to determinstically control the build, like CI systems and automation servers. There's no benefit to using this for local development and I wouldn't recommend it.

I'll update the readme with something mentioning CI.

Reply:
alright @jlmitch5 I've added some notes mentioning , CI servers, zuul, etc. and also a sentence about this not being intended for general development.